lms/djangoapps/verify_student/apps.py

```diff
@@ -0,0 +1,19 @@
+"""
+Student Identity Verification Application Configuration
+"""
+
+from django.apps import AppConfig
+
+
+class VerifyStudentConfig(AppConfig):
+    """
+    Application Configuration for verify_student.
+    """
+    name = 'verify_student'
+    verbose_name = 'Student Identity Verification'
+
+    def ready(self):
+        """
+        Connect signal handlers.
+        """
+        from .signals import handlers  # pylint: disable=unused-variable
```

Comment:
I think this could just be `import .signals`.  Am I seeing correctly that the signals module doesn't even have a `handlers` attribute?